sql/updates/world/2015_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE `creature_template` SET `minlevel`='80', `maxlevel`='80' WHERE `entry`='30230';
```

Comment:
Remove the ' ' characters from the numbers, then push to repo and see if Travis will run again.
```sql
UPDATE `creature_template` SET `minlevel`=80, `maxlevel`=80 WHERE `entry`=30230;